t/native_cp_al/conf/native_cp_al.conf.in

```diff
@@ -0,0 +1,7 @@
+config_version	1.2
+snapshot_root	@SNAP@
+cmd_rsync		@RSYNC@
+sync_first	1
+link_dest	0
+interval		hourly	6
+backup			@TEMP@	native_cp_al/
```

Comment:
We're testing that the / get automatically applied aren't we? So it should be missing in the config here

Reply:
That doesn't actually effect the failure mode. No harm in removing it. I will do that.